extractor/filesystem/language/python/pdmlock/extractor.go

```diff
@@ -0,0 +1,110 @@
+package pdmlock
```

Comment:
We'll need to add comments for all the other packages and public structs/functions in all the other files too.

You might want to make some of these private, e.g. pdmLockPackage/pdmLockFile since you're only using them inside the Extractor.

Reply:
Added docstrings to all public functions in the extractor, and made the rest private.